src/main/java/frc/robot/commands/ShooterBaseCommand.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.commands;
+
+
+
+import java.util.Map;
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.shooter.ShooterBase;
+
+public class ShooterBaseCommand extends Command {
+    ShooterBase shooter;
+    DoubleSupplier rightTriggerAxis;
+    ShuffleboardTab tab;
+    GenericEntry speedEntry;
+
+    public ShooterBaseCommand(ShooterBase shooter, DoubleSupplier rightTriggerAxis) {
```

Comment:
The righter doesn't need to be "rightTriggerAxis"....RobotContainer can tie this command to any button, so this should be something more generic, like "enable", "trigger", or something like that.

Reply:
It has been renamed to "enable".